### PR TITLE
Jobstate

### PR DESCRIPTION
**Pull request by lookarliu**
update testJobStateWithPaperJam

### CODE REVIEW

File: testcert.py [hunk 1]
```diff
@@ -3586,21 +3586,37 @@ def testJobStateWithPaperJam(self):
...
-    print 'Verify job is reported in error state.'
-    print 'Now clear the print path so the printer is no longer jammed.'
-    raw_input('Select enter once printer is clear of jam.')
-    print 'Verify print job prints after paper jam is cleared.'
-    try:
-      self.assertTrue(output)
-    except AssertionError:
-      notes = 'Error printing %s' % Constants.IMAGES['PDF9']
+    print 'Stick paper on the input tray which will cause a misfeed when feeding paper.'
+    raw_input('Select enter once it is sticked.')
```
Comment (kdlucas) [inline testcert.py]: Please change this to read:

raw_input('Select enter once paper is jammed.')

File: testcert.py [hunk 2]
```diff
@@ -3586,21 +3586,37 @@ def testJobStateWithPaperJam(self):
...
+      time.sleep(10)
+      job_state = gcpmgr.WaitJobStatusNotIn('printtest.pdf', ['Queued', 'In progress'])
+      try:
+        self.assertEqual(job_state, 'Error')
+      except AssertionError:
+        notes = 'Print Job is not in Error state: %s' % job_state
+        self.LogTest(test_id, test_name, 'Failed', notes)
+        raise
+      else:
+        print 'Wait for the print job to finish.'
```
Comment (kdlucas) [inline testcert.py]: Is the intention here to clear the paper jam and then let the print job finish? If so, we should place a comment in here that instructs the user to remove the paper jam.

**Comment by kdlucas** [comment]
Please see my comments and update.

**Comment by lookarliu** [comment]
I double check the case again and think the modification is not better than original codes. So this pull request should be closed.

**Pull request closed without merging**